### PR TITLE
ENH: Support for column specific col_space argument for frame.to_html…

### LINKED ISSUE #28917: DataFrame.to_html col_space parameter to change width of a specific column only
#### Problem description

Currently `col_space` changes width of all columns that are present in the dataframe. There must be a way to change the width of a specific column only while converting dataframe to html.

#### Expected Output
`col_space` to accept a list or dict where we can mention the column name to change only a specific column's width.

### PR DESCRIPTION
**Pull request by AakankshaAshok**
…() method (#28917)

- [ ] closes #28917
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] what's new: the col_space argument of to_html() can also take a dict or list to specify different col_space for different columns.

### CODE REVIEW

File: pandas/io/formats/format.py [hunk 1]
```diff
@@ -575,6 +575,16 @@ def __init__(
...
+
+        if isinstance(col_space, list) and len(col_space) != len(frame.columns):
+            raise ValueError(
+                (
+                    "Length of col_space list ({clen}) should match"
+                    " number of DataFrame columns ({dlen})".format(
+                        clen=len(col_space), dlen=len(frame.columns)
+                    )
+                )
+            )
```
Comment (gfyoung) [inline pandas/io/formats/format.py]: We'll be dropping Python 3.5 in `1.0`, so let's use F-string syntax

Comment (AakankshaAshok) [reply pandas/io/formats/format.py]: Hi, done. Does the message seem alright?

**Comment by WillAyd** [comment]
@AakankshaAshok is this still active? Can you add test(s)?

**Comment by AakankshaAshok** [comment]
> @AakankshaAshok is this still active? Can you add test(s)?

@WillAyd Yes. Will do. Any other suggestions?

**Comment by WillAyd** [comment]
Not for now; can file more comments after test(s) are added

**Comment by WillAyd** [comment]
@AakankshaAshok still active? If so please merge master

**Comment by MarcoGorelli** [comment]
Hi @AakankshaAshok  - sorry to chase you up, just wanted to ask whether you're still working on this :)

**Comment by WillAyd** [comment]
Closing as stale; ping if you'd like to pick back up

**Pull request closed without merging**